libsql-sqlite3/src/main.c

```diff
@@ -1480,6 +1480,10 @@ void sqlite3LeaveMutexAndCloseZombie(sqlite3 *db){
   if( db->lookaside.bMalloced ){
     sqlite3_free(db->lookaside.pStart);
   }
+
+  /* Destroy the create wal */
```

Comment:
nitpick: "Destroy the wal manager". But let's just drop this comment, the code below literally says the same thing :sweat_smile: